.ci/pytorch/macos-test.sh

```diff
@@ -244,13 +244,13 @@ test_torchbench_smoketest() {
       PYTHONPATH="$(pwd)"/torchbench python benchmarks/dynamo/torchbench.py \
         --performance --only "$model" --backend "$backend" --inference --devices "$device" "$dtype_arg" \
         --output "$TEST_REPORTS_DIR/inductor_${backend}_torchbench_${dtype}_inference_${device}_performance.csv" || true
-      if [ "$backend" == "inductor" ]; then
+      if [ "$backend" == "inductor" ] || [ "$backend" == "export-aot-inductor" ]; then
         PYTHONPATH="$(pwd)"/torchbench python benchmarks/dynamo/torchbench.py \
           --accuracy --only "$model" --backend "$backend" --inference --devices "$device" "$dtype_arg" \
           --output "$TEST_REPORTS_DIR/inductor_${backend}_torchbench_${dtype}_inference_${device}_accuracy.csv" || true
       fi
     done
-    if [ "$backend" == "inductor" ]; then
+    if [ "$backend" == "inductor" ] || [ "$backend" == "export-aot-inductor" ]; then
```

Comment:
Nit (not a bash expert, but there should be a way to pattern match that something ends with inductor in bash)
```suggestion
    if [[ "$backend" == *inductor ]]; then
```